src/Parser/Core/getCastEfficiency.js

```diff
@@ -64,6 +67,8 @@ export default function getCastEfficiency(CPM_ABILITIES, abilityTracker, combata
         overhealing,
         castEfficiency,
         recommendedCastEfficiency,
+        avgIssueCastEfficiency,
```

Comment:
I'd prefer spelling this out (average), consistent with the other props.

Reply:
Fixed